bool detected
geometry_msgs/Pose pose_hand

